node/src/components/linear_chain_sync.rs

```diff
@@ -614,6 +633,24 @@ where
                                 ),
                             );
                         }
+                        if block.protocol_version() != self.protocol_version {
+                            warn!(
+                                %peer,
+                                protocol_version = ?self.protocol_version,
+                                block_version = ?block.protocol_version(),
```

Comment:
`%` would probably look nicer than `?`? Not sure what they look like for protocol versions.

Reply:
Changed in [`339b851` (#1210)](https://github.com/CasperLabs/casper-node/pull/1210/commits/339b85175e29df8f895c3bdb0bbddf666ab0325e).